README.md

```diff
@@ -10,23 +10,44 @@ PX4 computer vision algorithms packaged as ROS nodes for depth sensor fusion and
 The active developers of the avoidance repo are syncing up on a bi-weekly basis on the WG dev call.
 
 * Join the call: https://zoom.us/j/506512712
-* Time: Monday, 9PM CET
+* Time: Monday, 5PM CET
 * Meeting ID: 506512712
 * Dronecode calendar: https://www.dronecode.org/calendar/
 
 [![PX4 Avoidance video](http://img.youtube.com/vi/VqZkAWSl_U0/0.jpg)](https://www.youtube.com/watch?v=VqZkAWSl_U0)
 
-# Quick Start with Docker
+# Table of Contents
+- [Getting Started](#getting-started)
+  - [Installation](#installation)
+    - [Quick Start with Docker](#quick-start-with-docker)
+    - [Installation for Ubuntu 16.04 and ROS Kinetic](#installation-for-ubuntu-16.04-and-ros-kinetic)
+  - [Run the Simulation](#run-the-simulation)
+    - [Local Planner](#local-planner)
+    - [Global Planner](#global-planner)
+  - [Run on Hardware](#run-on-hardware)
+    - [Prerequisite](#prerequisite)
+    - [Local Planner](#local-planner)
+    - [Global Planner](#global-planner)
+- [Troubleshooting](#troubleshooting)
+- [Advanced](#advanced)
+  - [Message Flows](#message-flow)
+    - [PX4 and local planner](#px4-and-local-planner)
+    - [PX4 and global planner](#px4-and-gloabl-planner)
+- [Contributing](#contributing)
+
+# Getting Started
+
+## Installation
+
+### Quick Start with Docker
 
 A ROS container based on Ubuntu 16.04 has been created and can be used to quickly try the simulation, as a demo. Running it is as simple as installing docker and docker-compose, and running `$ docker-compose up` from the right folder. Find the corresponding instructions [here](docker/demo).
 
 For __deployment__ instructions, check "[Deploying with Docker](docker#deploying-with-docker)".
 
 If you want to leverage docker in your __development__ environment, check the "[Developing with Docker](docker#developing-with-docker)" section.
 
-# Prerequisites
-
-## Installation for Ubuntu 16.04 and ROS Kinetic
+### Installation for Ubuntu 16.04 and ROS Kinetic
```

Comment:
I found that I was unable to run `rosrun` following these instructions. It might be necessary to install `rosbash` as well.